distributions/openhab/src/main/resources/bin/update.lst

```diff
@@ -134,6 +134,7 @@ ALERT;Twitter Binding has become X. All related references to Twitter and Tweets
 ALERT;AndroidTV Binding: The thing configuration of 'port' has been renamed and split to 'googletvPort' and 'shieldtvPort'
 ALERT;evcc Binding: Update to evcc API version 0.123.1 results in several new or updated channels. Existing Items will need to be adjusted.
 ALERT;ISM8 Binding: Most channels have changed and are now using Units of Measurements. Items must be adapted and the things must be recreated.
+ALERT;Jython Binding: Default python lib path changed from "/automation/lib/python" to "/automation/jython/lib" and default python script path changed from "automation/jsr223" to "automation/jython"
```

Comment:
Thanks for the update instruction. How should users deal with this change? Maybe instruct them to move any existing files from the old to the new dirs?